LabTools/Display/PlotDisplayWindow.py

```diff
@@ -564,8 +564,8 @@ def YCheckBoxHandler(self):
         """Update which data is used for the Y axis (both left and right)"""
         if self.chan_X is None:
             return
-        else:
-            print(self.chan_X)
+        #else:
```

Comment:
If you think it is not useful, then you can delete it rather than commenting it ;)